packages/bahmni-widgets/src/conditions/ConditionsTable.tsx

```
@@ -20,6 +22,7 @@ import {
 import { useQueryClient } from '@tanstack/react-query';
 import React, { useCallback, useMemo, useState } from 'react';
 import ConfirmationModal from '../confirmationModal/ConfirmationModal';
+import { useActivePractitioner } from '../activePractitioner';
```

ESLint

[error] `../activePractitioner` import should occur before import of `../confirmationModal/ConfirmationModal`

(import/order)